apps/perf-test/src/scenarioNames.js

```diff
@@ -1,16 +1,10 @@
 // You don't have to add scenarios to this structure unless you want their display name to differ from their scenario name.
 const scenarioNames = {
-  BaseButtonNew: 'BaseButton (experiments)',
-  DefaultButtonNew: 'DefaultButton (experiments)',
   DetailsRowFast: 'DetailsRow (fast icons)',
   DetailsRowNoStyles: 'DetailsRow without styles',
   DocumentCardTitle: 'DocumentCardTitle with truncation',
-  MenuButtonNew: 'MenuButton (experiments)',
-  PrimaryButtonNew: 'PrimaryButton (experiments)',
-  SplitButtonNew: 'SplitButton (experiments)',
   StackWithIntrinsicChildren: 'Stack with Intrinsic children',
   StackWithTextChildren: 'Stack with Text children',
-  ToggleNew: 'Toggle (experiments)',
```

Comment:
i think until we have a placement of VNext components, we should still keep them. Just that we really don't need to measure their perf for every CI build since they are not actively being developed.

Reply:
Another thing to remember is that perf-test isn't blocking or gating success/failure status on PRs. Even if it takes a few extra minutes, I think it's worth keeping them in as long as the components exist.